src/windows/common/WslClient.cpp

```diff
@@ -1554,13 +1556,15 @@ int WslaShell(_In_ std::wstring_view commandLine)
     parser.AddArgument(Integer(settings.MemoryMb), L"--memory");
     parser.AddArgument(Integer(settings.CpuCount), L"--cpu");
     parser.AddArgument(Utf8String(fsType), L"--fstype");
+    parser.AddArgument(testRootfs, L"--wsla-test-rootfs");
+    parser.AddArgument(containerRootVhd, L"--container-vhd");
     parser.AddArgument(help, L"--help");
     parser.Parse();
 
     if (help)
     {
         const auto usage = std::format(
-            LR"({} --wsla [--vhd </path/to/vhd>] [--shell </path/to/shell>] [--memory <memory-mb>] [--cpu <cpus>] [--dns-tunneling] [--fstype <fstype>] [--new-api] [--help])",
+            LR"({} --wsla [--vhd </path/to/vhd>] [--shell </path/to/shell>] [--memory <memory-mb>] [--cpu <cpus>] [--dns-tunneling] [--fstype <fstype>] [--wsla-test-rootfs] [--container-vhd </path/to/vhd>] [--help])",
```

Comment:
Now that we support the actual rootfs vhd with containerd, we should probably add the nuget package to the project and use that VHD by default. 

The easiest way I can think of doing this would be to propagate the nuget package path via a macro, and default the VHD path to that value (like we do for WSL_SYSTEM_DISTRO_PATH today)